### PR TITLE
feat: Add Query for Single Data Cert and List of Data Cert

### PR DESCRIPTION
**Pull request by inchori**
- Add cmd and keeper for query a single data cert and list of data cert.

### CODE REVIEW

File: x/datadeal/keeper/grpc_query_deal.go [hunk 1]
```diff
@@ -18,8 +21,55 @@ func (k Keeper) Deal(goCtx context.Context, req *types.QueryDealRequest) (*types
...
 }
+
+func (k Keeper) DataCert(goCtx context.Context, req *types.QueryDataCertRequest) (*types.QueryDataCertResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	dataCert, err := k.GetDataCert(ctx, req.DealId, []byte(req.DataHash))
```
Comment (hansol-medi) [inline x/datadeal/keeper/grpc_query_deal.go:37]: We may have to decode the string using base64 package. 
```suggestion
        bz, err := base64.StdEncoding.DecodeString(req.DataHash)
	if err != nil {
		return nil, err
	}
	dataCert, err := k.GetDataCert(ctx, req.DealId, bz)
```

**Review by gyuguen — APPROVED** [review]
LGTM 👍

**Review by hansol-medi — APPROVED** [review]
LGTM 👍

**Comment by inchori** [comment]
Query the list of data certificate has something wrong when I did manual test. So I'll convert to draft.

**Pull request merged**